crates/primitives-traits/src/block/recovered.rs

```diff
@@ -283,6 +283,15 @@ impl<B: Block> RecoveredBlock<B> {
         (self.block.into_block(), self.senders)
     }
 
+    /// Returns the `Recovered<&T>` transaction at the given index.
```

Comment:
Sender index?

Reply:
how senders are stored is effectively hidden so just index is okay here